### PR TITLE
`Communication`: Improve error message alert when entering a duplicate channel name

### PR DESCRIPTION
**Pull request by lennart-keller**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).
#### Server
- [x] **Important**: I implemented the changes with a very good performance and prevented too many (unnecessary) database calls.
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/server/).
- [x] I documented the Java code using JavaDoc style.
#### Client
- [x] **Important**: I implemented the changes with a very good performance, prevented too many (unnecessary) REST calls and made sure the UI is responsive, even with large data.
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

When renaming a channel on the messages tab and entering a name that already exists, the error message is very generic (Bad Request). 

### Description

The skipAlert field of the `ChannelNameDuplicateException` has now the value `false`. Therefore, the client now displays it properly. To avoid also displaying the "Bad Request" alert, I added a check, that prevents the generic alert from being shown if a more generic one has been already displayed due to `skipAlert=false`.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 General channels

1. Try to rename the first channel using the name of the other channel: An alert with a proper error message should be shown (see screenshots)
2. Switch the language: The alert should be properly translated
3. Rename the channel using a new name: No alert should be shown

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

#### Client

| Class/File | Line Coverage | Confirmation (assert/expect) |
|------------|--------------:|---------------------:|
| [conversation-info.component.ts](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5163/latest/artifact/shared/Coverage-Report-Client-Tests/app/overview/course-conversations/dialogs/conversation-detail-dialog/tabs/conversation-info/conversation-info.component.ts.html) | 90.69% | ✅ |
| [metis-conversation.service.ts](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5163/latest/artifact/shared/Coverage-Report-Client-Tests/app/shared/metis/metis-conversation.service.ts.html) | 84.02% | ✅ |

### Screenshots

before:
before

after:
after

### CODE REVIEW

**Review by DominikRemo — APPROVED** [review]
Tested on ts1. Works as expected following the testing steps.

Optional:
When creating a new channel with the same name, I get the "Bad Request" and "Channel with name [...] exists"
Bildschirmfoto 2023-07-11 um 19 02 25

**Review by DominikRemo — DISMISSED** [review]
Tested on ts1. Works as expected 👍
Thank you for addressing my feedback :)

File: src/main/java/de/tum/in/www1/artemis/service/metis/conversation/errors/ChannelNameDuplicateException.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ private static Map<String, Object> getParameters(String channelName) {
         Map<String, String> params = new HashMap<>();
         params.put("channelName", channelName);
         Map<String, Object> parameters = new HashMap<>();
-        parameters.put("skipAlert", true);
+        parameters.put("skipAlert", false);
```
Comment (JohannesStoehr) [inline src/main/java/de/tum/in/www1/artemis/service/metis/conversation/errors/ChannelNameDuplicateException.java:27]: Is false the default value here? If so, could you leave out this line?

Comment (lennart-keller) [reply src/main/java/de/tum/in/www1/artemis/service/metis/conversation/errors/ChannelNameDuplicateException.java]: By default the value is not set. I removed the line setting parameter to false.

**Review by DominikRemo — APPROVED** [review]

**Review by JohannesStoehr — APPROVED** [review]

**Review by MaximilianAnzinger — APPROVED** [review]
Code LGTM 👍

**Pull request merged**